src/fc/qemu/hazmat/ceph.py

```diff
@@ -315,11 +315,49 @@ def regen_xfs_uuid(self):
             self.cmd(f"xfs_db -x -c 'uuid generate' {partition}")
 
 
-class TmpSpec(VolumeSpecification):
-    suffix = "tmp"
-
+class SeedFCMixin:
     ENC_SEED_PARAMETERS = ["cpu_model", "rbd_pool"]
 
+    def seed_fc(self, enc, generation, compatibility_mode=False):
```

Comment:
I'd rather avoid the mixin. Just make a module-level function `seed_enc` and pass the spec object as the first parameter (not naming it `self`) and the rest can basically stay as is.